packages/twenty-server/src/core/open-api/utils/components.utils.ts

```diff
@@ -42,17 +42,16 @@ const getSchemaComponentsProperties = (
         itemProperty.type = 'boolean';
         break;
       case FieldMetadataType.RELATION:
-        itemProperty = {
-          type: 'array',
-          items: {
-            type: 'object',
-            properties: {
-              node: {
-                type: 'object',
-              },
+        if (field.fromRelationMetadata?.toObjectMetadata.nameSingular) {
```

Comment:
why are we changin this? what if the relation is in the other direction?

Reply:
We do that to remove "node" from relation fields in open api schemas (see images below). If the relation is in the other direction (many to one, one to one), we just need the RelationObjectId, which is already provided (eg: company in the nested object People)

Before 
<img width="465" alt="image" src="https://github.com/twentyhq/twenty/assets/29927851/37325e1e-c754-4ec9-8776-4b1a843d5d6a">

After 
<img width="632" alt="image" src="https://github.com/twentyhq/twenty/assets/29927851/54dd1483-ef71-4e20-ac5d-96c4a1353a0b">